src/js/utils/deviceFeaturesUtils.js

```diff
@@ -32,10 +32,12 @@ export const parseCapabilities = (features?: Features): Capability[] => {
     // needs to be "any" since Features.capabilities are declared as string[] but in fact it's a number[]
     const filter = (c: any) => (CAPABILITIES[c - 1] ? [CAPABILITIES[c - 1]] : []);
     // fallback for older firmware
-    if (!features.capabilities || !features.capabilities.length)
+    if (!features.capabilities || !features.capabilities.length) {
         return features.major_version === 1
             ? DEFAULT_CAPABILITIES_T1.flatMap(filter)
             : DEFAULT_CAPABILITIES_TT.flatMap(filter);
+    }
+
```

Comment:
nit/optional/food for though: This seems like a pure fomating refactor. I'm up for refactoring the code-base piece by piece as we work on it and I support this particular change as well. Let's maybe just think of separating the refactoring from feature implementation by commit each separately. That would have two benefits that go hand-in-hand:
-  easier review: I can put all my focus on the crucial feature-implementing changes and quickly skim over the formatting refactoring commits.
- more refactoring: if we commit the refactoring separately we don't have to worry about it cluttering the new code. We can then boldly refactor whole files in a PR knowing the reviewer can still fully focus on the new code.

Reply:
I will remove it